.dotnet/src/Custom/FineTuning/FineTuningClient.cs

```diff
@@ -13,38 +14,41 @@ namespace OpenAI.FineTuning;
 [CodeGenSuppress("CreateFineTuningJob", typeof(FineTuningOptions))]
-[CodeGenSuppress("GetPaginatedFineTuningJobsAsync", typeof(string), typeof(int?))]
-[CodeGenSuppress("GetPaginatedFineTuningJobs", typeof(string), typeof(int?))]
-[CodeGenSuppress("RetrieveFineTuningJobAsync", typeof(string))]
+[CodeGenSuppress("ListPaginatedFineTuningJobsAsync", typeof(string), typeof(int?), typeof(CancellationToken))]
+[CodeGenSuppress("ListPaginatedFineTuningJobs", typeof(string), typeof(int?))]
+[CodeGenSuppress("RetrieveFineTuningJobAsync", typeof(string), typeof(CancellationToken))]
 [CodeGenSuppress("RetrieveFineTuningJob", typeof(string))]
-[CodeGenSuppress("CancelFineTuningJobAsync", typeof(string))]
+[CodeGenSuppress("CancelFineTuningJobAsync", typeof(string), typeof(CancellationToken))]
 [CodeGenSuppress("CancelFineTuningJob", typeof(string))]
-[CodeGenSuppress("GetFineTuningEventsAsync", typeof(string), typeof(string), typeof(int?))]
-[CodeGenSuppress("GetFineTuningEvents", typeof(string), typeof(string), typeof(int?))]
-[CodeGenSuppress("GetFineTuningJobCheckpointsAsync", typeof(string), typeof(string), typeof(int?))]
-[CodeGenSuppress("GetFineTuningJobCheckpoints", typeof(string), typeof(string), typeof(int?))]
+[CodeGenSuppress("ListFineTuningEventsAsync", typeof(string), typeof(string), typeof(int?), typeof(CancellationToken))]
+[CodeGenSuppress("ListFineTuningEvents", typeof(string), typeof(string), typeof(int?))]
+[CodeGenSuppress("ListFineTuningJobCheckpointsAsync", typeof(string), typeof(string), typeof(int?), typeof(CancellationToken))]
+[CodeGenSuppress("ListFineTuningJobCheckpoints", typeof(string), typeof(string), typeof(int?))]
 public partial class FineTuningClient
 {
-    // CUSTOM: Remove virtual keyword.
-    /// <summary>
-    /// The HTTP pipeline for sending and receiving REST requests and responses.
-    /// </summary>
-    public ClientPipeline Pipeline => _pipeline;
-
     // CUSTOM: Added as a convenience.
-    /// <summary> Initializes a new instance of <see cref="FineTuningClient">. </summary>
+    /// <summary> Initializes a new instance of <see cref="FineTuningClient"/>. </summary>
     /// <param name="apiKey"> The API key to authenticate with the service. </param>
     /// <exception cref="ArgumentNullException"> <paramref name="apiKey"/> is null. </exception>
     public FineTuningClient(string apiKey) : this(new ApiKeyCredential(apiKey), new OpenAIClientOptions())
     {
     }
 
+    // CUSTOM: Added as a convenience.
+    /// <summary> Initializes a new instance of <see cref="FineTuningClient"/>. </summary>
+    /// <param name="apiKey"> The API key to authenticate with the service. </param>
+    /// <param name="options"> The options to configure the client. </param>
+    /// <exception cref="ArgumentNullException"> <paramref name="apiKey"/> is null. </exception>
+    public FineTuningClient(string apiKey, OpenAIClientOptions options) : this(new ApiKeyCredential(apiKey), options)
```

Comment:
We already have a constructor with `ApiKeyCredential credential` parameter. Why do we need another one with a `string apikey` parameter?